### DISCUSSION TITLE
Clustering support?

### DISCUSSION THREAD

**Discussion by charanjeet123** [jobrunr/jobrunr#461] [Q&A] [answered]
We had following requirements and I am glad to say that JobRunr almost fulfils all, except the clustering which I couldn't be sure after reading the documentation, as it is not clearly mentioned.
1. Job Scheduling (Persistency/Scheduled)
2. Cron Support
3. Free to use
4. Triggers
5. NoSQL (MongoDB)

Does JobRunr supports clustering like Quartz (through property org.quartz.jobStore.isClustered) http://www.quartz-scheduler.org/documentation/quartz-2.3.0/configuration/ConfigJDBCJobStoreClustering.html?

**Comment by dmurphy-ambra** [comment, marked as answer]
Jobrunr supports clustering.  The diagram at the top of the [documentation page](https://www.jobrunr.io/en/documentation/) makes this clear.

Clustering does not need to be explicitly enabled, you simply run two or more instances of your service and point them at the same storage provider instance (e.g. database), and Jobrunr will automatically distribute the jobs across all the instances of your service connected to that storage provider.
